Tests/scripts/configure_tests.py

```diff
@@ -14,14 +14,35 @@
 CONTENT_DIR = os.path.abspath(SCRIPT_DIR + '/../..')
 sys.path.append(CONTENT_DIR)
 
-from Tests.scripts.constants import *  # noqa: E402
-from Tests.test_utils import get_yaml, str2bool, get_from_version, get_to_version, \
+from demisto_sdk.commands.common.constants import *  # noqa: E402
+from demisto_sdk.commands.common.tools import get_yaml, str2bool, get_from_version, get_to_version, \
     collect_ids, get_script_or_integration_id, run_command, LOG_COLORS, print_error, print_color, \
     print_warning  # noqa: E402
 
 # Search Keyword for the changed file
 NO_TESTS_FORMAT = 'No test( - .*)?'
-
+PACKS_SCRIPT_REGEX = r'{}/([^/]+)/{}/(script-[^\\/]+)\.yml$'.format(PACKS_DIR, SCRIPTS_DIR)
+FILE_IN_INTEGRATIONS_DIR_REGEX = r'{}/(.+)'.format(INTEGRATIONS_DIR)
+FILE_IN_SCRIPTS_DIR_REGEX = r'{}/(.+)'.format(SCRIPTS_DIR)
+FILE_IN_PACKS_INTEGRATIONS_DIR_REGEX = r'{}/([^/]+)/{}/(.+)'.format(
+    PACKS_DIR, INTEGRATIONS_DIR)
+FILE_IN_PACKS_SCRIPTS_DIR_REGEX = r'{}/([^/]+)/{}/(.+)'.format(
+    PACKS_DIR, SCRIPTS_DIR)
+INTEGRATION_REGEXES = [
+    INTEGRATION_REGEX,
+    BETA_INTEGRATION_REGEX,
+    PACKS_INTEGRATION_REGEX
+]
+INCIDENT_FIELD_REGEXES = [
+    INCIDENT_FIELD_REGEX,
+    PACKS_INCIDENT_FIELDS_REGEX
+]
```

Comment:
why aren't those defined in the SDK?

Reply:
they're only used here, we can move them after dean's changes

Reply:
weird...
no need for now.